sdk/servicebus/Azure.Messaging.ServiceBus/samples/TopicFilters/TopicFilters.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
```

Comment:
Do we have a strategy for ensuring that the stand-alone samples are built and smoke tested with nightly runs?

Reply:
Hmm, I would have assumed that these would get built automatically already based on the folder structure, but I will have to double check.

Reply:
It seems to not be hooked up correctly. I will create an issue to address this in all the samples that we create.

Reply:
Created https://github.com/Azure/azure-sdk-for-net/issues/20842 to track this.